pepys_admin/view_data_cli.py

```diff
@@ -74,6 +74,24 @@ def do_view_table(self):
             table = selected_table[:-3] + "y"
         else:
             table = selected_table[:-1]
+
+        if table == "alembic_version":
```

Comment:
@BarisSari - don't we have a hard-coded version of this table name?

Reply:
Added: https://github.com/debrief/pepys-import/pull/457/commits/e7f571512941f5210accb5eddca87a2642ba6229
Note: I didn't change the names above because they will be changed when #427 is merged.